README.md

```diff
@@ -0,0 +1,129 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>To-DoList</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Fullstack Development ](#-fullstack-development-)
+  - [🛠  Built With ](#--built-with-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+
+# 📖 Fullstack Development <a name="To-Do List"></a>
+
+**Orgnised your time with to-do list** 
+
+## 🛠  Built With <a name="built-with"></a>
+- HTML
+- SASS
+- JavaScript
+- React
+### Key Features <a name="key-features"></a>
+- implement a function for adding a new task (add a new element to the array).
+- Implement a function for deleting a task (remove an element from the array).
+- Implement a function for editing task descriptions.
+- By default new tasks have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task is equal to 5).
+- Deleting a task update all remaining items' indexes, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) is set to 1).
+- All changes to the To Do List is saved in local storage.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://sagieramos.github.io/react-todo-app)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+### Setup
+
+Clone this repository to your desired folder:
+```sh
+  git clone https://github.com/sagieramos/react-todo-app $HOME
+```
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  cd $HOME/react-todo-app
+  npm install
+  npm run build
+  npm run dev
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+👤 ***Stanley Osagie Ramos**
+- GitHub: [@sagieramos](https://github.com/sagieramos)
+- Twitter: [@sagieramos](https://twitter.com/sagieramos)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/sagieramos)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Improve the UI**
+
```

Comment:
In the "Future Features" section, consider providing more details on what improvements you plan to make in the UI. This can help potential contributors understand your vision better.